file-formats/advc/README.md

```diff
@@ -0,0 +1,5 @@
+# ADVC File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :--- | :--- | :--- | :---
+`<name>.advc.json` | 1 | [`zif_aff_advc_v1.intf.abap`](./type/zif_aff_advc_v1.intf.abap) | [`advc-v1.json`](./advc-v1.json) | [`z_aff_example_advc.advc.json`](./examples/z_aff_example_advc.advc.json)
```

Comment:
Since the File Format for ADVC is not yet complete I will talk to my colleagues whether we want to remark this here in the readme. I will come back to you with a suggestion for this remark.

Reply:
I will take the task of finding a suitable sentence offline and update this README accordingly in a new pull request. I am going to approve this pull request